src/components/banners/Banner.tsx

```diff
@@ -26,6 +26,12 @@ export type BannerProps = {
   showLessLinkText?: ReactNode;
   // color for show more links
   showMoreLinkColor?: string;
+  // banner margin, padding, text font size
+  spacing?: {
+    margin?: string,
+    padding?: string,
+  };
+  fontSize?: string;
```

Comment:
Hey DK, what do you think about making these `string | number` types, or maybe better yet, `CSSProperties['margin']`, etc?

Reply:
@chowington Thank you for your review! I agree that using CSSProperties is better. Changed them accordingly 👍